tests/test_meshtastic_async_utils.py

```
@@ -191,6 +192,142 @@ def test_action_succeeds(self):
+            patch.object(
+                mu,
+                "_record_ble_teardown_timeout",
+                side_effect=_record_timeout_with_worker_finish,
+            ) as mock_record,
+            patch.object(
+                mu,
+                "_resolve_ble_teardown_timeout",
+                wraps=real_resolve_timeout,
+            ) as mock_resolve,
+        ):
+            with pytest.raises(TimeoutError):
```

Ruff (0.15.10)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)